client/components/domains/use-my-domain/style.scss

```diff
@@ -83,6 +83,14 @@
 			}
 		}
 	}
+
+	& &__domain-input-fieldset::before {
+		content: "https://";
+		color: #000;
```

Comment:
We should use color variables so that Calypso colour themes continue work.

Reply:
Replaced with `$black` from `@wordpress/base-styles/colors`

Reply:
Hum, that seems fine, although I had expected CSS variable instead of Sass variable `var( --some-variable-for-text )`